test/parallel/test-http-response-multiheaders.js

```diff
@@ -47,7 +48,7 @@ const server = http.createServer(function(req, res) {
 });
 
 server.listen(0, common.mustCall(function() {
-  let count = 0;
+  const countdown = new Countdown(2, () => server.close());
   for (let n = 1; n <= 2; n++) {
```

Comment:
Could we store `2` in a const or something? Potentially this could also be rewritten to utilize the countdown more fully (loop could be based on countdown being > 0, etc.) but at the very least we should remove magic numbers.

Reply:
@apapirovski might be a good idea. Do you have any variable name ideas? Maybe `runs = 2` or `runCount = 2`?

Reply:
Either of those sound good 👍

Reply:
@apapirovski here is the commit https://github.com/nodejs/node/pull/17419/commits/a80a5433f80c08f139fb9b2e425ec4a5244014ca